hive/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -0,0 +1,104 @@
+ * under the License.
+ */
+
+package org.apache.iceberg.hive;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.hadoop.hive.metastore.api.Database;
+import org.apache.iceberg.AssertHelpers;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.exceptions.NoSuchNamespaceException;
+import org.apache.thrift.TException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestHiveCatalog extends HiveMetastoreTest {
+
+  private static Map meta = new HashMap<String, String>() {
+    {
+      put("owner", "apache");
+      put("group", "iceberg");
+      put("comment", "iceberg  hiveCatalog test");
+    }
+  };
+
+  @Test
+  public void testCreateNamespace() throws TException {
+    Namespace namespace = Namespace.of("dbname");
+
+    catalog.createNamespace(namespace, meta);
+
+    Database database = metastoreClient.getDatabase(namespace.toString());
+    Map<String, String> dbMeta = catalog.getMetafrpmhiveDb(database);
+    Assert.assertTrue(dbMeta.get("owner").equals("apache"));
+    Assert.assertTrue(dbMeta.get("group").equals("iceberg"));
+    Assert.assertEquals("there no same location for db and namespace",
+        dbMeta.get("location"), catalog.nameSpaceToHiveDb(namespace, meta).getLocationUri());
+  }
+
+  @Test
+  public void testListNamespace() throws TException {
+    Namespace namespace1 = Namespace.of("dbname1");
+    catalog.createNamespace(namespace1, meta);
+
+    Namespace namespace2 = Namespace.of("dbname2");
+    catalog.createNamespace(namespace2, meta);
+    List<Namespace> namespaces = catalog.listNamespaces();
+    Assert.assertTrue("hive db not hive the namespace 'dbname1'", namespaces.contains(namespace1));
+    Assert.assertTrue("hive db not hive the namespace 'dbname2'", namespaces.contains(namespace2));
+  }
+
+  @Test
+  public void testLoadNamespaceMeta() throws TException {
+    Namespace namespace = Namespace.of("dbname_load");
+
+    catalog.createNamespace(namespace, meta);
+
+    Map<String, String> nameMata = catalog.loadNamespaceMetadata(namespace);
+    Assert.assertTrue(nameMata.get("owner").equals("apache"));
+    Assert.assertTrue(nameMata.get("group").equals("iceberg"));
+    Assert.assertEquals("there no same location for db and namespace",
+        nameMata.get("location"), catalog.nameSpaceToHiveDb(namespace, meta).getLocationUri());
+  }
+
+  @Test
+  public void testDropNamespace() throws TException {
+
+    Namespace namespace = Namespace.of("dbname_drop");
+    catalog.createNamespace(namespace, meta);
+
+    Map<String, String> nameMata = catalog.loadNamespaceMetadata(namespace);
+    Assert.assertTrue(nameMata.get("owner").equals("apache"));
+    Assert.assertTrue(nameMata.get("group").equals("iceberg"));
+
+    Assert.assertTrue("drop namespace " + namespace.toString() + "error", catalog.dropNamespace(namespace, false));
+    AssertHelpers.assertThrows("should throw exception", NoSuchNamespaceException.class,
```

Comment:
The assertion messages help identify what is being tested. "Should throw exception" isn't quite specific enough. How about "Should fail to drop when namespace doesn't exist"